lms/djangoapps/teams/static/teams/js/views/teams_tab.js

```diff
@@ -1,14 +1,28 @@
 ;(function (define) {
     'use strict';
 
-    define(['backbone', 'underscore', 'text!teams/templates/teams-tab.underscore'],
-        function (Backbone, _, teamsTabTemplate) {
-            var TeamTabView = Backbone.View.extend({
-                render: function() {
-                    this.$el.html(_.template(teamsTabTemplate, {}));
-                }
-            });
+    define(['backbone',
+            'underscore',
+            'gettext',
+            'js/components/header/views/header',
+            'js/components/header/models/header',
+            'text!teams/templates/teams-tab.underscore'],
+           function (Backbone, _, gettext, HeaderView, HeaderModel, teamsTabTemplate) {
+               var headerModel = new HeaderModel({
+                   description: gettext("Course teams are organized into topics created by course instructors. Try to join others in an existing team before you decide to create a new team!"),
```

Comment:
Good catch, @dan-f. I think this is placeholder doc. @srpearce, could you review the text strings in this PR when you have a moment.